packages/core/src/analytics/hubspotClient.ts

```diff
@@ -0,0 +1,63 @@
+import { Client } from "@hubspot/api-client";
+import { ApiException } from "@hubspot/api-client/lib/codegen/crm/contacts/apis/exception";
+
+const accessToken = process.env.HUBSPOT_ACCESS_TOKEN;
+if (!accessToken) {
+  throw new Error("Missing HUBSPOT_ACCESS_TOKEN");
+}
+
+const hubspotClient = new Client({ accessToken });
+
+interface CreateHubspotCustomerInput {
+  email: string;
+  firstName: string | null;
+  lastName: string | null;
+  marketingAccepted: boolean;
+}
+
+export const createHubspotCustomer = async ({
+  email,
+  firstName,
+  lastName,
+  marketingAccepted,
+}: CreateHubspotCustomerInput) => {
+  let id: string | undefined;
+  try {
+    const result = await hubspotClient.crm.contacts.basicApi.getById(
+      email,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
```

Comment:
I feel like they could change the signature here!

Reply:
Yeah, I'm actually a little unimpressed by that package in general. For example, it doesn't export their error instances